pkg/logs/processor/processor.go

```diff
@@ -113,6 +119,13 @@ func (p *Processor) run() {
 				log.Errorf("Error while reconfiguring the SDS scanner: %v", err)
 				order.ResponseChan <- err
 			} else {
+			    p.flareCtl.SetStandardSDSRules([]string{})
+			    p.flareCtl.SetEnabledSDSRules([]string{})
+			    if p.sds.IsReady() {
+					p.flareCtl.SetStandardSDSRules(p.sds.GetStandardRulesNames())
+					p.flareCtl.SetEnabledSDSRules(p.sds.GetEnabledRulesNames())
+				    p.flareCtl.SetSDSLastReconfiguration(time.Now())
```

Comment:
Since all three functions are always called together, would it make sense to refactor this as `SetSDSInfo(standard_rules, enabled_rules, time)` and `ClearSDSInfo`?

Reply:
Good idea, will do this, thanks.